apps/admin-gui/src/app/shared/components/members-candidates-list/members-candidates-list.component.html

```diff
@@ -25,14 +25,14 @@
         <td *matCellDef="let memberCandidate" class="static-column-size align-checkbox" mat-cell>
           <div
             [matTooltip]="getTooltip(memberCandidate) | translate"
-            [matTooltipDisabled]="!isCheckboxDisabled(memberCandidate)"
+            [matTooltipDisabled]="!(memberCandidate | disabledCandidate: blockManualAdding)"
```

Comment:
you can create new pipe for the get tooltip also, to get rid of the function in the html